config/initializers/1_fast_gettext.rb

```diff
@@ -15,6 +15,11 @@
 # work in all domains context by default (for plugins)
 include FastGettext::TranslationMultidomain
 
+# Keep TRANSLATORS comments
+Rails.application.config.gettext_i18n_rails.xgettext = %w[--add-comments=TRANSLATORS:]
+# Disable fuzzy .po merging
+Rails.application.config.gettext_i18n_rails.msgmerge = %w[--no-fuzzy-matching]
```

Comment:
gettext 3.2.1 fixes the issue, I've updated the minimum dependency to get the fix.